Kconfig

```diff
@@ -69,9 +69,14 @@ menu "LVGL configuration"
 
 	endmenu
 
+	config LV_USE_CLIB
+		bool
+		default n
+
 	menu "Memory settings"
 		choice
 			prompt "Malloc functions source"
+			default LV_USE_CLIB_MALLOC if LV_USE_CLIB
```

Comment:
There will be 2 defaults if `LV_USE_CLIB` ?

Reply:
We could make the 2nd one depend on !LV_USE_CLIB, but IIRC Kconfig picks the first default value which matches the conditions. So leaving the last one without any conditions should be fine.